bigtop-packages/src/common/bigtop-select/install_select.sh

```diff
@@ -67,12 +77,37 @@ LIB_DIR=${LIB_DIR:-/usr/lib/bigtop-select}
 BIN_DIR=${BIN_DIR:-/usr/bin}
 CONF_DIR=${CONF_DIR:-/etc/bigtop-select/conf.dist}
 
-STACK_ROOT_DIR=/usr/bigtop
+STACK_ROOT_DIR=$(echo ${PARENT_DIR} | sed -e "s/\/${BIGTOP_BASE_VERSION}$//")
 STACK_SELECTOR=distro-select
 CONF_SELECTOR=conf-select
 
 # Install packages
 install -d -p -m 755 $PREFIX${LIB_DIR}/
-install -d -p -m 755 $PREFIX${STACK_ROOT_DIR}/
+install -d -p -m 755 $PREFIX${PARENT_DIR}/
 install -p -m 755 ${DISTRO_DIR}/${STACK_SELECTOR} $PREFIX${LIB_DIR}/
 install -p -m 755 ${DISTRO_DIR}/${CONF_SELECTOR} $PREFIX${LIB_DIR}/
+cat > $PREFIX${LIB_DIR}/params.py <<EOF
+#!/usr/bin/env python2
```

Comment:
I prefer python3 to python2 since the latter has already been EOL'd. Also, would you include the dependency on python3 in the spec file?

Reply:
> I prefer python3 to python2 since the latter has already been EOL'd. Also, would you include the dependency on python3 in the spec file?

Sorry I just saw it.
@sekikn  I can't get rid of python2, because ambari only supports python2, and I think they need to be consistent.

Reply:
OK I understand. Thank you for the elaboration.